go.mod

```diff
@@ -168,7 +168,6 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zipkinreceiver v0.83.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zookeeperreceiver v0.83.0
 	go.opentelemetry.io/collector v0.83.0
-	go.opentelemetry.io/collector/confmap v0.83.0
```

Comment:
why this change?

Reply:
Most likely because the module has its own go.mod, so this dependency is no longer needed to be applied from the root go.mod.